node/examples/electron/image_classification/main.js

```diff
@@ -18,14 +18,22 @@ function createWindow() {
     }
   })
 
-  // Load the index.html with 'numRuns' to run inference multiple times.
+
   let url = `file://${__dirname}/image_classification/index.html`
+  let numRunsParam, deviceParam
```

Comment:
There are some duplicated codes for these samples, should we combine them into one function like ParseAndCheckParams()?

Reply:
We are going to migrate all standalone webnn samples into one folder to reduce duplicated codes, https://github.com/otcshare/webnn-native/issues/796

So let's optimize this there.

Reply:
OK,got it, thanks!